Exiled.API/Features/Generator.cs

```diff
@@ -224,10 +224,8 @@ public KeycardPermissions KeycardPermissions
         /// </summary>
         /// <param name="scp079Generator">The <see cref="Scp079Generator"/> instance.</param>
         /// <returns>A <see cref="Generator"/> or <see langword="null"/> if not found.</returns>
-        public static Generator Get(Scp079Generator scp079Generator) =>
-            Scp079GeneratorToGenerator.TryGetValue(scp079Generator, out Generator generator) ?
-            generator :
-            new(scp079Generator);
+        public static Generator Get(Scp079Generator scp079Generator) => scp079Generator == null ? null :
```

Comment:
Does TryGetValue() not just return null if it's given a null scp079Generator value?

Reply:
![image](https://user-images.githubusercontent.com/66829532/232768732-2fce9329-5d4f-4618-8daa-d01fd7917e3e.png)
nope it's will return NRE